app/views/projects/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%= @project.name %></h1>
+
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @project.challenge.theme %></p>
+<p>Number of Contestants: <%= @project.contestants.count %></p>
+<p>Average Contestant Experience: <%= Contestant.avg_experience %></p>
+
+
+<h3>Add New Contestant to the <%= @project.name %> Challenge</h3>
+<%= form_tag "/projects/#{@project.id}" do %>
```

Comment:
We talked about this in the review, but just going to put it here too for consistency ;) This route could cause a conflict when your application wants to implement functionality for creating a project. Instead, maybe consider handrolling a route specific for this case, or use the `/contestant_projects` route, and send the project id through with query params